index.js

```diff
@@ -1 +1 @@
- 
\ No newline at end of file
+module.exports = require('./src/main/pom-parser/pom-parser');
```

Comment:
Given that we might want to have additional APIs on this project other than just pom parsing, we might export an object from this that has a property of POMParser.

The usage would probably be then:
```js
const POMParser = require('java-utils').POMParser;

POMParser.read...(...);
```